### PR TITLE
WiP: Debian packaging

### PR DESCRIPTION
**Pull request by mih**
This is not asking for a merge. The purpose of this PR is to document the state of the effort, allow for comments and track related issues.

Related issues:
- [x] #1470
- [ ] #1472
- [ ] #1473
- [ ] #1476
- [x] #1477
- [x] #1478
- [ ] #1480
- [ ] #1481
- [ ] #1511
- [ ] #1689 
- [ ] #1690
- [ ] #1691

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -21,8 +21,6 @@ Package: jasp
 Architecture: any
 Depends: ${shlibs:Depends},
          ${misc:Depends},
-         libqt5webkit5,
-         libqt5svg5,
```
Comment (akashrajkn) [inline debian/control]: libqt5webkit5 and libqt5svg5 are required, am I right?

Comment (mih) [inline debian/control]: Yes, but they are discovered automatically.

**Comment by mih** [comment]
FTR: Binary packages for Debian sid and stretch, as well as Ubuntu xenial are now available from NeuroDebian.

Backports to earlier releases require backports of rinside, which in turn require backports of R 3.3. Doable but requires some effort.

**Comment by boutinb** [comment]
Hi @mih: thanks really much for your work!
If I well understood your work, you have made a package under NeuroDebian of JASP Beta 0.8.2. And Debian & Ubuntu can use this package to install JASP.
2 questions:
Is there a way to update this package when we have a new release?
Should we merge this pull request now?

**Comment by mih** [comment]
@boutinb Yes that is right.

You could ping me for an update when you make a release. I'll build binary packages and update the NeuroDebian repo.However, there is no secret sauce. We could have a (virtual) sit-down to demonstrate how you could update and build packages on your own.

You could consider merging this into a separate branch, e.g. 'debian'. Merging it into master will cause more trouble than it help in the long run. Often the distributions will require things that make less sense or are needlessly complicated to achieve in the general case -- when it is straightforward for a particular environment.

**Comment by mih** [comment]
@boutinb One more thing:

The folder `debian/patches` contains a few patches that I would very much appreciate to appear in master, though!

**Comment by boutinb** [comment]
@mih Cannot you use a define NEURODEBIAN or something like that, and use it directly in the code? In this way, the patches files are not necessary anymore.
We can make also a folder debian/neurodebian and debian/ubuntu, so that we can keep both versions in the same repository. Or do you think this is a bad idea?
To have a (virtual) sit-down would be really much appreciated! Have you any plan to come to Amsterdam? Eric-Jan Wagenmaker asked you once, but we don't know if you answered him. If not, we can arrange a Skype session.
We would like really much to have a sustainable port of JASP on the linux platforms!

**Comment by boutinb** [comment]
@mih I have just discussed with Eric-Jan, you are more than welcome in Amsterdam if you would like. We are ready to pay you back travel and hotel costs. If not, would you prefer a Skype session?

**Comment by mih** [comment]
@boutinb Sorry for the delay, missed the github notification. I would love to come, but the semester just started and experience tell that a little later will result in less penalty for leisure activities ;-)

**Comment by boutinb** [comment]
@mih When you have time, you would more than welcome!

**Comment by boutinb** [comment]
@mih What should we do with this request? Can I close it?

**Comment by mih** [comment]
Pushed binary packages of 0.8.1.0 (amd64) for Debian stretch/sid and Ubuntu 16.04 to NeuroDebian.

**Comment by a-0-p** [comment]
Hi @mih, if you have time could you spend a couple of words on how to properly install jasp on Debian 9? I see that it's on the neurodebian repositories, however when trying to install it I have multiple missing or broken dependencies...for example r-cran-afex which is needed to run jasp is not available on Debian repositories. If you could guide less experienced users with the installation, that'll be nice.

Thanks!

**Comment by a-0-p** [comment]
A quick update: it turned out that once I have installed the package r-cran-afex from the unstable repositories, there are no more problems in running jasp on Debian. Generally, I would avoid installing packages from unstable into stable; however, r-cran-afex is very unlikely to give any problems.

So that's what worked for me...but in case anyone has any suggestion or any other workaround to install jasp on Debian 9, I think it's worth sharing them here.

Thanks!

**Comment by boutinb** [comment]
@mih I think we can close this pull request now. We use FlatPack now, and it works on Debian (and on most Linux versions).

**Pull request closed without merging**